source/docs/articles/sites/domains.md

```diff
@@ -33,12 +33,29 @@ You can simultaneously add both the bare domain name and the www subdomain. This
 ### Develop Using a Domain Without Changing DNS
 Sometimes it's useful to develop on a site using a specific domain, but the overhead of temporarily changing DNS is too much. Use the following workaround to allow your local workstation to access your Pantheon site by the desired domain without changing DNS. This requires a paid plan.
 
-1. From the Pantheon Dashboard, add the domain to the target site environment.
-2. Add a line to your local <a href="https://en.wikipedia.org/wiki/Hosts_(file)">Hosts file</a> with the provided IP address, default value for the Live environment, and the bare domain.
+<div class="alert alert-danger" role="alert">
+<h4>Important</h4>This process is for short-term DNS testing. Endpoint IPs will change over time. Be sure to use the Dashboard's recommendations and provided IPs when going Live.</div>
+
+1. From the command line, dig for the Pantheon IPs for your Live Pantheon URL.
+
+Example:
+```
+dig +short live-yoursite.pantheon.io
+```
+You should receive something similar to the following output:
+```
+styx-xx.pantheon.io.
+192.123.456.789
+104.123.456.78
+23.123.45.6
+```
+
+2. From the Pantheon Dashboard, add your domain to the target site environment.
+3. Add a line to your local <a href="https://en.wikipedia.org/wiki/Hosts_(file)">Hosts file</a> with one of the IP addresses revealed via dig, default value for the Live environment, and the bare domain.
 
 Example:
 ```
-192.237.142.203 live-example.pantheon.io example.com
+192.123.456.789 live-yoursite.pantheon.io yoursite.com
 ```
 
 Remove this entry when you're ready to configure DNS.
```

Comment:
Add note to follow Site Dashboard recommendations when going Live, just to emphasize that these test settings should not be used to configure DNS?

Also wondering if it'd be helpful to make the "Remove this entry..." step 4.